### ISSUE TITLE
A11y_AzureDataStudio_AddNewController_AI4D : Name property does not exists for "Remember Password" checkbox

### ISSUE THREAD

**Issue by 12-shweta** [microsoft/azuredatastudio#7659]
"[Check out Accessibility Insights!](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier."
​
GitHubTags:#A11y_AzureDataStudio;#A11yMAS;#A11yTCS;#WCAG1.3.1;WCAG4.1.2;#WCAG4.2.1;#DesktopApp;#NVDA;#JAWS;#AI4D;#SQL Azure Data Studio;#Win32

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 6582debd7388b1be512fc3a9b956d3e54f6115e7
Date: 2019-10-03T08:56:27.454Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362
NVDA Version: 2019.2
JAWS Version:2019.1907.42 ILM

**Additional Details:**
Tool Name : AI4D

**Steps to Reproduce:**
1. Install and Launch the application the azure data studio application.
2. Connect to the servers with the required credentials.
3. Navigate to the "SQL Server Big Data Cluster" and expand it.
4. Navigate to the "SQL Server Big Data Cluster controller" and click on it.
5. Open AI4D and check that name property exists for "Remember Password" checkbox

**Actual:**
Name property does not exists for "Remember Password" checkbox.

**Expected:**
Name property should be provided for "Remember Password" checkbox

**User Impact:**
Screen reader user will it difficult to understand the purpose of the control.

**Attachment for Reference:**
![7659_A11y_AzureDataStudio_AddNewController_AI4D](https://user-images.githubusercontent.com/54656883/66633011-727a2d00-ec27-11e9-9f78-b2285f8a53ed.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by 12-shweta** [comment]
Verified the bug in below environment :
Version: 1.15.0-insider (user setup)
Commit: 041f34b
Date: 2020-01-08T05:36:19.161Z
VS Code: 1.38.0
Electron: 6.1.6
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Windows_NT x64 10.0.18363
Issue is fixed and does not reproduce .
![7659_Closed](https://user-images.githubusercontent.com/54656883/72725257-4584d100-3bab-11ea-8697-96672ae83080.png)

**Comment by fsteffi** [comment]
#Azure Data Studio - New Features_win32;

**Comment by Rama8345** [comment]
A11ySev2

**Comment by Samhitha66** [comment]
#Closed; #Regressed:07-14-23;
